app/views/jobs/show.html.erb

```diff
@@ -8,3 +8,21 @@ Level of Interest: <%= @job.level_of_interest %>
 <br>
 
 Company: <%= link_to @job.company.name, company_path(@job.company) %>
+<br>
+
+<%= link_to "Edit", edit_company_job_path %>
+<%= link_to "Delete", company_job_path, method: :delete %>
+<%= link_to "Jobs Index", company_jobs_path %>
+<h3>Comments</h3>
+<%= form_for @comment do |f| %>
+  <%= f.label :comment %>
+  <%= f.text_field :content %>
+  <%= hidden_field(:job, :id) %>
```

Comment:
i know we talked about this in class, but if we nested our resources it would eliminate the need to pass the hidden field here. if this is still confusing, we can talk about it early next week 👍